ethereum/p2p/src/test/java/org/hyperledger/besu/ethereum/p2p/network/P2PPlainNetworkTest.java

```diff
@@ -87,8 +87,8 @@ public void closeVertx() {
   @Test
   public void handshaking() throws Exception {
     final NodeKey nodeKey = NodeKeyUtils.generate();
-    try (final P2PNetwork listener = builder("compa", "avocado").nodeKey(nodeKey).build();
-        final P2PNetwork connector = builder("compb", "banana").build()) {
+    try (final P2PNetwork listener = builder("partner1client1").nodeKey(nodeKey).build();
```

Comment:
What does it mean that we've gone from a list of two items ("compa", "avocado") to a one item ("partner1client1")?

Reply:
this simply maps to the directory in the resources folder that contains the certificates. I modified it to take two parameters when I changed the directory structure to `compa/client`. Now that I have reverted the directory structure as it was before this PR, we only need to specify single parameter.